test/giphify_api_web/controllers/search_controller_test.exs

```diff
@@ -0,0 +1,5 @@
+defmodule GiphifyApiWeb.SearchControllerTest do
+  use GiphifyApiWeb.ConnCase
+
```

Comment:
I left this blank because I couldn't get my test to pass. You can view my failing test below. 
I don't understand how to test a create action. I've read the HexDocs [Controllers](https://hexdocs.pm/phoenix/testing_controllers.html) which provide great examples for index, and show actions but nothing else.  Essentially I just want to assert that a 201 status is returned if everything goes smoothly! The sad path would test for a :error, and the correct response.

`defmodule GiphifyApiWeb.SearchControllerTest do
  use GiphifyApiWeb.ConnCase
  import Mox

  alias GiphifyApi.Search
  alias GiphifyApi.Query

  @create_attrs %{query: "Cat in Hat"}

  def fixture(:query) do
    {:ok, query} = Query.create_search(@create_attrs)
    query
  end

  setup %{conn: conn} do
    {:ok, conn: put_req_header(conn, "accept", "application/json")}
  end

  describe "create greeting" do
    test "renders greeting when data is valid", %{conn: conn} do
      GiphifyApi.GiphySearch.MockClient
      |> expect(:get_gif, fn "Dog" ->
        "https://itworks.com"
      end)

      conn = post conn, search_path(conn, :create), @create_attrs
      assert %{"id" => id} = json_response(conn, 201)["data"]
    end
  end
end`